dev/release/rat_exclude_files.txt

```diff
@@ -129,3 +129,5 @@ Cargo.lock
 .history
 parquet-testing/*
 *rat.txt
+datafusion/proto/src/generated/pbjson.rs
```

Comment:
There is precedent in arrow for excluding generated files - https://github.com/apache/arrow-rs/blob/master/dev/release/rat_exclude_files.txt#L20